src/h5web/vis-packs/core/shared/Mesh.tsx

```diff
@@ -15,7 +15,7 @@ function Mesh(props: Props) {
     abscissaScale,
     ordinateScale,
     visSize,
-  } = useContext(AxisSystemContext);
+  } = useAxisSystemContext();
```

Comment:
Unrelated but wow that I think about it, I think we should rename `Mesh` to `VisMesh` in our code.

Reply:
Ah thought you had done it. Yeah, definitely.